### PR TITLE
SDK - Refactoring - Split the K8sHelper class

### PR DESCRIPTION
**Pull request by Ark-kun**
One part was only used by container builder and provided higher-level API over K8s Client.
Another was used by the compiler and did not use the kubernetes library.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2333)

### CODE REVIEW

File: sdk/python/kfp/containers/_k8s_job_helper.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# Copyright 2018 Google LLC
```
Comment (numerology) [inline sdk/python/kfp/containers/_k8s_job_helper.py]: nit: 2019

Comment (Ark-kun) [reply sdk/python/kfp/containers/_k8s_job_helper.py]: Done

**Pull request merged**